backend/src/api/jwt.rs

```diff
@@ -35,9 +36,25 @@ pub(crate) async fn jwt(
             if !user.can_upload(&context.config.auth) {
                 deny!("upload");
             }
-            Ok(context.jwt.new_token(Some(&user), json!({
-                "roles": ["ROLE_STUDIO"],
-            })))
+            if let Some(opencast_id) = opencast_id {
+                let key = format!("e:{opencast_id}");
+
+                // TODO: remove once OC18 is released
+                let read_role = format!("ROLE_EPISODE_{}_READ", opencast_id);
+                let write_role = format!("ROLE_EPISODE_{}_WRITE", opencast_id);
+
+                Ok(context.jwt.new_token(Some(&user), json!({
+                    "oc": {
+                        key: ["read", "write"],
+                    },
+                    // TODO: remove once OC18 is released
+                    "roles": [read_role, write_role],
```

Comment:
This should still include `ROLE_STUDIO` as well, as otherwise the user might not have access to the API at all.